src/components/AssetsTable/index.jsx

```diff
@@ -332,6 +346,12 @@ export class AssetsTable extends React.Component {
     this.setState({
       deletedAsset: {},
       deletedAssetIndex: null,
+      statusAlertOpen: false,
```

Comment:
I'm not sure that this suggestion is better - just something I thought of. Would it be worth it to have some kind of variable 'defaultStatusAlertState' that we could refer to here? That way, we could just replace these lines with { ...defaultStatusAlertState}, and use that in the constructor as well. I'm just a little worried it won't be super clear that we're just resetting back to the defaults in this setState call. The use of the comment just makes me wonder if we could try to be a little bit more clear here. Maybe even pulling the setState out into a function 'resetStatusState'? I don't feel super strongly about it - just throwing it out there! :)

Reply:
Would it be enough to rename the existing function to `clearStatusAlert`?

Reply:
Looking at this again, I think the function clears and closes, so I think it's probably fine as is, in my opinion. Thanks for entertaining the idea, though.